yt/data_objects/region_expression.py

```diff
@@ -136,13 +142,13 @@ def _slice_to_edges(self, ax, val):
         return l, r
 
     def _create_region(self, bounds_tuple):
-        left_edge = []
-        right_edge = []
+        left_edge = self.ds.domain_left_edge
```

Comment:
So I am pretty sure that we set `domain_left_edge` to be a "read-only" property right now, but it would make me feel a lot better if this were done via copies.

Reply:
you got it !